message/schemas/xregistry_openapi_messagedefinition.json

```diff
@@ -1,11 +1,15 @@
 {
-  "openapi": "3.0.6",
+  "openapi": "3.1.0",
   "info": {
     "title": "xRegistry API",
     "description": "xRegistry API",
     "version": "0.5-wip"
   },
-  "servers": [],
+  "servers": [
+    {
+      "url": "https://raw.githubusercontent.com/clemensv/spec-1/fix-schemas/message/schemas/"
```

Comment:
should this be pointing to an xreg URL?

Reply:
I will revert that bit.

Reply:
https://github.com/swagger-api/swagger-editor/issues/4241